deepwell/migrations/20220906103252_deepwell.sql

```diff
@@ -101,6 +101,20 @@ ALTER TABLE site
     ADD CONSTRAINT site_custom_domain_fk
     FOREIGN KEY (custom_domain) REFERENCES site_domain(domain);
 
+--
+-- Site Membership
+--
+
+CREATE TABLE site_member (
+    membership_id BIGSERIAL PRIMARY KEY,
+    user_id BIGSERIAL NOT NULL REFERENCES "user"(user_id),
+    site_id BIGSERIAL NOT NULL REFERENCES site(site_id),
```

Comment:
`BIGSERIAL` is for primary keys, since it creates an autoincrement counter. Here `user_id` and `site_id` are not getting assigned new values, they're foreign keys to an existing table. These should be `BIGINT`.

Reply:
Wasn't aware, I'll fix